guides/Identities.md

```diff
@@ -0,0 +1,78 @@
+# About Identities
+
+API calls are restricted using identities. An identity defines who is calling a method and what actions they are authorized to take. 
+Each method can only be called by specific identities. [Notes in the REST reference](#methods-with-restricted-identities) indicate restrictions on the identities that can call a method.  
+Different contexts also limit the identities that can make a given call. For example, apps can only make calls as apps, not as visitors or members.
```

Comment:
Ah, you mean apps vs. headless?

Reply:
yes. I basically pulled in some info that should really be in an "auth strategies" article (like we have in the SDK) b/c it felt too confusing otherwise

Reply:
Didnt understand what the context means - you need to describe it